string Class
uint32[] Center
uint32 Left
uint32 Top
uint32 Right
uint32 Bottom
